core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/CarbonUtils.java

```diff
@@ -108,6 +109,10 @@ public class CarbonUtils {
     private static boolean isServerConfigInitialized;
     private static Log audit = CarbonConstants.AUDIT_LOG;
     private static Gson gson = new Gson();
+    private static final HashMap<String, String> fileExtensions = new HashMap<String, String>() {{
+        put("JKS", ".jks");
+        put("PKCS12", ".p12");
+    }};
```

Comment:
since the extension types are fixed, we can use an enum here

Reply:
let's add the enum in the keystoreUtils and make it public

Reply:
Addressed with commit b995db3d535f7cb6004a9a49575bd550c10e4be6